kvbc/src/v4blockchain/v4_blockchain.cpp

```diff
@@ -61,6 +61,13 @@ KeyValueBlockchain::KeyValueBlockchain(
   }
 }
 
+KeyValueBlockchain::~KeyValueBlockchain() {
+  if (snap_shot_) {
+    auto tmp_snap_shot = RecoverySnapshot{&native_client_->rawDB()};
+    tmp_snap_shot.releasePreviousSnapshot(snap_shot_);
```

Comment:
Why not call directly, one-liner and doesn't create this tmp object:
`native_client_->rawDB().ReleaseSnapshot(snap_shot_);`
or
```
const auto& db = native_client_->rawDB();
db.ReleaseSnapshot(snap_shot_);
```

Reply:
thanks, done